templates/alert.html

```diff
@@ -1,6 +1,8 @@
+<style><% include ../assets/css/cd-alert.css %></style>
+
 <% if (typeof alert !== 'undefined' && alert.type && alert.message) { %>
-  <div class="alert alert--inline alert--<%= alert.type %> [ flow ]" role="alert">
-    <div class="alert__inner">
+  <div class="cd-alert cd-alert--inline cd-alert--<%= alert.type %> [ flow ]" role="alert">
+    <div class="cd-alert__inner">
```

Comment:
Same for the `.cd-alert__inner` — if your reference markup isn't using it (and I son't see the class in the official component) then we can cut it from HID.

Reply:
I attempted to remove this wrapper element, but it does seem to be necessary in order to allow the yet-to-be-added icon to appear in the correct location. When I removed the container, various block-level tags started displaying in a row and changing the class to be `cd-alert__message` seems like the easy change here, so that it both means something, and leaves room for the icon to be added.